tests/bricks/test_conv.py

```diff
@@ -259,14 +229,14 @@ def test_convolutional_sequence():
     batch_size = 5
     activation = Rectifier().apply
 
-    conv = ConvolutionalLayer(activation, (3, 3), 5,
-                              (pooling_size, pooling_size),
-                              weights_init=Constant(1.),
-                              biases_init=Constant(5.))
+    conv = ConvolutionalActivation(activation, (3, 3), 5,
```

Comment:
I do not understand why an activation needs `weights_init` and `biases_init`. Moreover, I do not understand why it accepts these arguments and does not crash.